media-api/app/lib/elasticsearch/filters.scala

```diff
@@ -36,6 +36,9 @@ object filters {
   def term(field: String, term: String): FilterBuilder =
     termFilter(field, term)
 
+  def term(field: String, value: Boolean): FilterBuilder =
```

Comment:
Might be better to change the signature of the `term` above to `field: String, term: Object` as the [es library supports it](https://github.com/elastic/elasticsearch/blob/v1.3.4/src/main/java/org/elasticsearch/index/query/FilterBuilders.java#L127).